source/stylesheets/base/_buttons.scss

```diff
@@ -41,3 +42,13 @@ $_button-background-color-hover: shade($action-color, 20%);
     }
   }
 }
+
+.secondary {
+  background-color: transparent;
+  border: 2px solid $action-color;
+  color: $action-color;
+}
+
+.top-space {
```

Comment:
I'd break this out into it's own component file. Maybe it's in a partial called `_utility.scss` or `_spacing.scss`?